receiver/receiverhelper/go.mod

```diff
@@ -0,0 +1,61 @@
+module go.opentelemetry.io/collector/receiver/receiverhelper
+
+go 1.23.0
+
+require (
+	github.com/stretchr/testify v1.10.0
+	go.opentelemetry.io/collector/component v1.27.0
+	go.opentelemetry.io/collector/component/componenttest v0.120.0
+	go.opentelemetry.io/collector/pipeline v0.121.0
+	go.opentelemetry.io/collector/receiver v0.120.0
```

Comment:
We probably need something like check-module-version in core collector to catch this